index.js

```diff
@@ -108,17 +124,18 @@ function sniff(buffer, callback) {
     var zlib_opts = {finishFlush: zlib.Z_SYNC_FLUSH };
     zlib_gunzip(buffer, zlib_opts, function (err, output) {
         if (err) return callback(invalid('Unknown filetype'));
-        returnOutput(output,callback);
+        returnOutput(output, callback);
     });
 }
 
-function waft(buffer, callback) {
+function getProtocol(type) {
     var mapping = {
         csv: 'omnivore:',
         mbtiles: 'mbtiles:',
         shp: 'omnivore:',
         zip: 'omnivore:',
         tif: 'omnivore:',
+        "tif+gz": 'omnivore:',
```

Comment:
This wasn't here prior, which means we had `undefined` returned when a tif+gz file was passed in and we wanted the protocol back.